c-cannons/bangazon-initial_site
# order history and details

#### Descriptions

Can view all previous orders and the details for each individual order.

#### Steps to Test

```
git fetch origin ml-order-history
git checkout origin ml-order-history
```

1. Go to the user profile
2. Click on the order history link
3. Should be presented with a table of previous orders including the date and payment method showing the last 4 digits of the account number
4. Click on 'Details'
5. Should be presented with a table displaying the quantity of each product, product name, product price and the total price of the order

#### Link to Feature Ticket

#16
